installer/windows/build_installer.py

```diff
@@ -64,6 +81,10 @@ def build_massastation():
         ],
         check=True,
     )
+    os.rename(
+        os.path.join("..", "cmd", "massastation", "MassaStation.exe"),
```

Comment:
here why it's `MassaStation.exe` and not var `MASSASTATION_BINARY`?

Reply:
When we run `fyne package`, the `-name` correspond to the name of the package we want to make. In the case of Windows, the resulting package is a `.exe` named as the `-name` option, so in that case, `MassaStation.exe`. That's why we need to move it, and rename it.

Reply:
okay, you can add a comment in the code explaining that.

Reply:
Done